holmes/plugins/toolsets/newrelic/new_relic_api.py

```
@@ -0,0 +1,119 @@
+            url,
+            headers=headers,
+            json=graphql_query,
+            timeout=timeout,
+        )
+        response.raise_for_status()
+
+        # Parse JSON response
+        data = response.json()
+
+        # Check for GraphQL errors even on 200 responses
+        if "errors" in data and data["errors"]:
```

Ruff (0.12.2)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)